### ISSUE TITLE
Monogrammed Sash does not drop

### ISSUE THREAD

**Issue by zhuk90** [TeamEverlook/Everlook-BugTracker#161]
Mob named Mok'rash has no drop of mentioned item. I've killed him thrice.

https://classicdb.ch/?item=3985

On every server i've played it had like 100% drop.

**Comment by red-candle** [comment]
Yeah, the first time we killed him, it didn't drop. So we were in a raid and even though non-raid quests update in raids on this server, I thought that perhaps the reason for the missing drop was the raid.

The second time we killed him, no raid and it dropped for all.
